tempesta_fw/http.c

```diff
@@ -1055,6 +1146,7 @@ tfw_http_resp_process(TfwConnection *conn, struct sk_buff *skb,
 			TFW_INC_STAT_BH(serv.msgs_parserr);
 			return TFW_BLOCK;
 		case TFW_POSTPONE:
+			((TfwHttpResp *)hmresp)->skb_offset = off;
```

Comment:
Yes, it **is** a workaround. The offset is needed only in the case when the body of a response is terminated by closing of the connection. There's no other case so far where the offset is need after the skb it belongs to has been processed.

The last skb is always the last one in the list for a message, so it's easily reachable anyway, so there's no need to keep a duplicate pointer to it elsewhere. I think. As for the offset, that might be a good idea.